unicsv.cc

```diff
@@ -1228,15 +1204,15 @@ unicsv_parse_one_line(char* ibuf)
 static void
 unicsv_rd()
 {
-  char* buff;
+  QString buff;
 
   if (unicsv_fieldsep == nullptr) {
     return;
   }
 
-  while ((buff = gbfgetstr(fin))) {
-    buff = lrtrim(buff);
-    if ((*buff == '\0') || (*buff == '#')) {
+  while ((buff = fin->readLine(), !buff.isNull())) {
```

Comment:
As above.

Reply:
same as above, no auto conversion from QString to bool.